src/testdrive/src/parser.rs

```diff
@@ -567,6 +581,7 @@ impl<'a> Iterator for LineReader<'a> {
                 }
                 let pos = self.pos;
                 self.pos += i;
+                self.raw_pos += i + 1;
```

Comment:
didn't fully read through this logic, but if `raw_pos` is supposed to be the byte position then you'll need to account for non-ASCII characters. Instead of `+ 1` you can do `+ c.len_utf8()`

Reply:
The character here is always `\n`, so a single byte. I'll add a comment.